components/gc-dwnld/download_link-en.html

```diff
@@ -0,0 +1,54 @@
+---
+{
+	"title": "Download link example",
+	"language": "en",
+	"altLangPage": "download_link-fr.html",
+	"breadcrumbs": [
+		{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/index-en.html" }
+	],
+	"dateModified": "2023-04-20"
+}
+---
+<div class="wb-prettify all-pre"></div>
+
+<h2>Download link</h2>
```

Comment:
This heading level 2 are redondent with the heading level 1 which are render as "Download link example" as specified in the front-matter.

```suggestion
```